environment.yml

```diff
@@ -8,16 +8,15 @@ channels:
   - conda-forge
 dependencies:
   - python=3.9 #need to pin to avoid issues with builds
-  - cython>=0.29.30
+  - cython>=3.0
   - compilers
   - openblas
   - nomkl
   - setuptools=59.2.0
-  - meson >= 0.64.0
```

Comment:
this one is good indeed, it's a dependency of meson-python and that won't change.

Reply:
Do you mean "good, it is a dependency and so this line is not needed" or "good, please revert the removal"?

Reply:
sorry, "good it is indeed not needed". I'll resolve this one.